packages/tools/devtools/devtools-view/src/components/PermissionModal.tsx

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import { Button, Checkbox, makeStyles, shorthands, tokens } from "@fluentui/react-components";
+import { useTelemetryOptIn } from "../TelemetryUtils.js";
+
+const PermissionModalStyles = makeStyles({
```

Comment:
We've been using this naming pattern... I'm not sure if one or the other is more appropriate, so let's stay consistent here.
```suggestion
const useStyles = makeStyles({
```